src/index.js

```diff
@@ -5,17 +5,20 @@ import App from "./App";
 import { makeServer } from "./server";
 import {BrowserRouter as Router} from "react-router-dom";
 import { FilterProvider } from "./context/FilterContext";
+import { CartProvider } from "./context/CartContext";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <FilterProvider>
-      <Router>
-        <App />
-      </Router>
-    </FilterProvider>
+    <CartProvider>
+      <FilterProvider>
+        <Router>
+          <App />
+        </Router>
+      </FilterProvider>
+    </CartProvider>
```

Comment:
you can make a Allprovider like as you router

Reply:
ok will do that.